### PR TITLE
chore: Minor code cleanup

### PR DESCRIPTION
**Pull request by michaelgrigoryan25**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ results.json
 lib/kubernetes_config_*
 tests/assets/eks-options.json
 tests/assets/do-options.json
+Dockerfile
```
Comment (benjaminch) [inline .gitignore:14]: Not sure we need it?

File: src/container_registry/docr.rs [hunk 2]
```diff
@@ -305,7 +309,7 @@ impl ContainerRegistry for DOCR {
     fn push(&self, image: &Image, force_push: bool) -> Result<PushResult, EngineError> {
         let registry_name = self.get_registry_name(image)?;
 
-        let _ = match self.create_repository(&image) {
+        let _ = match self.create_repository(image) {
```
Comment (benjaminch) [inline src/container_registry/docr.rs:312]: Is `let _ =` still needed in this context?

File: src/container_registry/ecr.rs [hunk 3]
```diff
@@ -249,19 +249,19 @@ impl ECR {
                     ),
                 ))
             }
-            _ => Ok(self.get_repository(&image).unwrap()),
+            _ => Ok(self.get_repository(image).unwrap()),
```
Comment (benjaminch) [inline src/container_registry/ecr.rs:252]: OPT: replace `unwrap()` with an `expect("cannot get repository")`

**Comment by benjaminch** [comment]
Thanks a lot for your contribution @michaelgrigoryan25 :)

File: .gitignore (hunk 1 above)
Comment (michaelgrigoryan25) [reply .gitignore:14]: I am developing in a Docker container on Windows 10 that's why I added a Dockerfile to `.gitignore`

File: src/container_registry/docr.rs (hunk 2 above)
Comment (michaelgrigoryan25) [reply src/container_registry/docr.rs:312]: Was wondering the same thing. Maybe not? 🤔

File: src/container_registry/ecr.rs (hunk 3 above)
Comment (michaelgrigoryan25) [reply src/container_registry/ecr.rs:252]: Will do after I get home 👍

File: src/container_registry/docr.rs (hunk 2 above)
Comment (michaelgrigoryan25) [reply src/container_registry/docr.rs:312]: Will patch 👍

**Comment by michaelgrigoryan25** [comment]
Patched everything @benjaminch!

**Review by benjaminch — APPROVED** [review]
LGTM, I will merge it soon, thanks :)

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -45,7 +45,7 @@ jobs:
           sccache --show-stats
           echo "########## START UNIT TESTS ##########"
           cargo test
-  selected-functionnal-tests:
+  selected-functional-tests:
```
Comment (benjaminch) [inline .github/workflows/tests.yml:48]: Do not update this one, it requires a change on our end as well, better to do it in another PR.

Comment (michaelgrigoryan25) [reply .github/workflows/tests.yml:48]: Sure

**Comment by michaelgrigoryan25** [comment]
Any updates on this?

**Comment by benjaminch** [comment]
Should be good, I will merge it today
